frontend/src/App.vue

```diff
@@ -173,11 +174,15 @@ export default {
          && !this.isDone);
     },
     back() {
+      this.gotoStep(1);    // go 1 step back
+    },
+    gotoStep(numOfSteps) { // go numOfSteps step back
```

Comment:
for me back function name is ok
I do not think we need a new function called gotoStep
back function default arg should be 1
You even use invoke with back parameter

Reply:
next phase is enabling going fwd, so gotostep can get -value or +value.
agreed with Barlev.